pkg/models/model_plan.go

```diff
@@ -4,13 +4,24 @@ import "github.com/google/uuid"
 
 // ModelPlan is the top-level object for an entire draft model plan
 type ModelPlan struct {
-	BaseStruct
+	baseStruct
 	ModelName string      `json:"modelName" db:"model_name"`
 	Archived  bool        `json:"archived" db:"archived"`
 	Status    ModelStatus `json:"status" db:"status"`
 }
 
-// GetModelPlanID returns the modelPlanID of the task list section
+//NewModelPlan returns a new model Plan
```

Comment:
A bit pedandic, perhaps, but could you just add details of the defaults that this uses (and perhaps other constructors?)

These function comments are nice for auto-complete, and seeing something along the lines of `NewModelPlan returns a new unarchived model plan with a default status of ModelStatusPlanDraft` might be helpful!

Reply:
I like it, I think you are right. It was so quick to implement, I just did it really quick, as well as updating the comment on plan discussion.